.github/workflows/phpstan/autoload.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types = 1);
```

Comment:
Maybe we should put this file into a package we can reuse ? 🤔  (it could be `php-dev-tools` or another*)

Else we will have to copy it into the sixty (and more !) native modules, and if we need to update it at some point, it'll need 60 (and more !) PRs 😭 

(there is a [tool](https://github.com/matks/prestashop-repos-bulk-editor) to update modules in bulk, but still 😛 )

*`prestashop/phpstan-setup` ?

Reply:
Bad idea to apply to each module. Because PHPStan will be red so you create the PR but all modules will be to fix ;)

I will fix them at each release.

Reply:
@matks It can be a good idea : `prestashop/phpstan-config` like `prestashop/stylelint-config`  & `prestashop/eslint-config`

Reply:
This suggestion has already been taken in account: We have a common `autoload.php` which is available in `prestashop/php-dev-tools`

You should this phpstan.neon as a base: https://github.com/PrestaShop/php-dev-tools/blob/master/templates/phpstan/phpstan.neon